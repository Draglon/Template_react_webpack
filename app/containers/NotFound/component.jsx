import React from 'react';

const NotFoundComponent = () => (
  <section className="section not-found">
    <div className="wrapper">
      <h2 className="not-found__title">Page not found</h2>
      <h1 className="not-found__subTitle">404</h1>
    </div>
  </section>
);

export default NotFoundComponent;
