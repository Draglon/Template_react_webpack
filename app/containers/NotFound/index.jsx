import React from 'react';

const NotFound = ({ translate: t }) => (
  <section className="section not-found">
    <div className="wrapper">
      <h2 className="not-found__title">{t.title}</h2>
      <h2 className="not-found__subTitle">{t.subTitle}</h2>
    </div>
  </section>
);

export default NotFound;
