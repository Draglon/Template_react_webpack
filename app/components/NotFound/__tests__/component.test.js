import React from 'react';
import toJson from 'enzyme-to-json';

import NotFound from '../component';

describe('<NotFound /> component', () => {
  const component = shallow(<NotFound />);

  it('render property', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
