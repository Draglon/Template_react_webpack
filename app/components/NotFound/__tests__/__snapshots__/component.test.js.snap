// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> component render property 1`] = `
<section
  className="section not-found"
>
  <div
    className="wrapper"
  >
    <h2
      className="not-found__title"
    >
      Page not found
    </h2>
    <h1
      className="not-found__subTitle"
    >
      404
    </h1>
  </div>
</section>
`;
